topi/include/topi/transform.h

```diff
@@ -57,7 +57,7 @@ using namespace topi::detail;
 inline Tensor expand_dims(const Tensor& x,
                           int axis,
                           int num_newaxis = 1,
-                          std::string name = "tensor",
+                          std::string name = "expand_dims",
```

Comment:
Why do some names start with the prefix `T_` and some don't?

Reply:
The `T_` prefix is used to avoid name conflicts with possible built-in functions.
Some are missing. Fixed!